### ISSUE TITLE
Readme on Docker-Compose Postgres edit

### ISSUE THREAD

**Issue by samschlicht** [talentbeyondboundaries/tbbtalentv2#1662]
Currently our readme instructs to run containerised postgresql using docker-compose, but then departs to give instructions to install locally (which is not needed), verify that it is installed locally, and populate the local DB.

Suggest we change the local instructions to docker ones and also perhaps a brief description how to connect IntelliJ.

**Comment by samschlicht** [comment]
**(Starts from after the docker-compose section of readme, which is all good)**

Verify that Docker is up and running, incl. containerised Postgres: 
`docker ps`

Verify that tctalent is up and running within that container:
`docker exec -it docker-compose-postgres-1 psql -U tctalent -d tctalent`
should open the psql console — `/q` to exit the psql console at any time.

**For existing dev**
Make a dump of the containerised DB:
`docker exec -it docker-compose-postgres-1 pg_dump --file=/tmp/dump.sql --create --username=tctalent --host=localhost --port=5432`
Copy the file from the docker container to local system:
`docker cp docker-compose-postgres-1:/tmp/tcdump.sql </path/to/file>`
Send to new dev

**New dev**
Copy the DB dump from your local system to your docker container:
`docker cp <path/to/file> docker-compose-postgres-1:/tmp/dump.sql`
Populate the docker DB from the dump:
`docker exec -it docker-compose-postgres-1 psql -U tctalent -d tctalent -f /tmp/dump.sql`

Connect IntelliJ to the docker DB:
Database > New > Data Source > PostgreSQL
1. Give the DB a name that distinguishes it clearly from Staging and Prod DBs that you later may add
2. Populate the other parameters with the default values in the postgres configuration of project file docker-compose.yml

**Optional:** for existing devs to understand that they no longer need to have postgres installed locally
`brew uninstall postgresql`

**Comment by sadatmalik** [comment]
Thanks for running through these steps, @samschlicht. The original issue #1403 suffered from massive scope creep! What was originally going to be simply updating the Readme with Redis docker instructions became adding Elasticsearch and Kibana services to the project and Readme, then Postgres too, and then it ultimately morphed into updating to Node 18 and Angular 16.

Despite my notes to return to the Readme and take a walk-through before asking everyone to cut over to the Postgres docker service, you beat me to it! Which is actually a great help. Thanks so much, Sam!

I agree with all your suggestions. Please go ahead and update the file at your convenience.

**Comment by samschlicht** [comment]
Yes I figured this one had just got lost in the midst of all that good work, @sadatmalik! I'll assign myself to this issue and edit the readme accordingly, probably later today. 😎
